content/en/guides/integrations/keras.md

```diff
@@ -258,7 +258,7 @@ See our [example repo](https://github.com/wandb/examples) for scripts, including
 
 The `WandbCallback` class supports a wide variety of logging configuration options: specifying a metric to monitor, tracking of weights and gradients, logging of predictions on training_data and validation_data, and more.
 
-Check out [the reference documentation for the `keras.WandbCallback`]({{< relref "/ref/python/integrations/keras/wandbcallback.md" >}}) for full details.
+Check out the reference documentation for the `keras.WandbCallback` for full details.
```

Comment:
How do they get to the reference documentation now to check it out?

Reply:
They won't. They'll need to use __doc__ (dunder method) to see it.  This was per request of the SDK Team because these APIs are some combination of broken and/or not maintained.

Reply:
Got it, as long as they will understand the new call to action here.